sycl/source/detail/graph_impl.hpp

```diff
@@ -1120,6 +1120,8 @@ class graph_impl : public std::enable_shared_from_this<graph_impl> {
     return MBarrierDependencyMap[Queue];
   }
 
+  unsigned long long getID() { return MID; }
+
```

Comment:
Think this function can be const, also applies to same function in other classes.